batch/batch/server/server.py

```diff
@@ -301,7 +301,8 @@ def __init__(self, pod_spec, batch_id, attributes, callback, parent_ids,
         else:
             self.refresh_parents_and_maybe_create()
 
-    def refresh_parents_and_maybe_create(self):
+    # pylint incorrect error: https://github.com/PyCQA/pylint/issues/2047
+    def refresh_parents_and_maybe_create(self):  # pylint: disable=invalid-name
```

Comment:
you could also redefine the function name regex in pylintrc. I somewhat prefer that. It's called `function-rgx`.